qiita_db/processing_job.py

```diff
@@ -591,6 +591,9 @@ def create(cls, user, parameters, force=False):
                         TTRN.add(sql, [artifact_info, job_id])
                     else:
                         pending[artifact_info[0]][pname] = artifact_info[1]
+                elif pname == 'artifact':
```

Comment:
Would you mind describing what's going on here?

Reply:
Sure, the `create_command` method in patch `58.py` adds the `delete_artifact`  and other Qiita internal commands; however, the artifact_id is in the parameter "artifact" vs. the parameter description (*); thus, if we want to add these commands to the database we need this special case. 

(*) all regular jobs, except internal Qiita, have their parameters in this format: parameter_name: [parameter_type, value]; so normally, we just check that parameter_type is `artifact` but for internal jobs we need to check the actual parameter.